e2e/guards-instances/index.ts

```diff
@@ -9,7 +9,7 @@ import {
 import { createApp, ref, reactive, defineComponent, computed } from 'vue'
 
 // override existing style on dev with shorter times
-if (!__CI__) {
+if (!window.__CI__) {
```

Comment:
This should be doable with https://vitejs.dev/config/#define

Reply:
Yes it is

Reply:
I have recovered